apps/docs/src/examples/xr/use-hand-joint/JointBody.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang='ts'>
+  import { useFrame } from '@threlte/core'
+  import { handJoints, useHandJoint } from '@threlte/xr'
+  import type { RigidBody as RapierRigidBody } from '@dimforge/rapier3d-compat'
+  import { Collider, RigidBody } from '@threlte/rapier'
+
+  export let jointIndex: number
+  export let hand: 'left' | 'right'
+
+  let body: RapierRigidBody
+
+  const joint = useHandJoint(hand, handJoints[jointIndex]!)
+
+  $: radius = joint.current?.jointRadius ?? 0.0001
```

Comment:
make `joint` reactive:

```suggestion
$: radius = $joint?.jointRadius ?? 0.0001
```